File: Dataset/Leetcode/train/14/279.java
 class Solution {
    public String XXX(String[] strs) {
        if(strs==null||strs.length==0){
            return "";
        }
        if(strs.length==1){
            return strs[0];
        }
        Arrays.sort(strs);
        int len=Math.min(strs[0].length(),strs[strs.length-1].length());
        for(int i=0;i<len;++i){
            if(strs[0].charAt(i)==strs[strs.length-1].charAt(i)){
                continue;
            }else{
                return strs[0].substring(0,i);
            }
        }
        return strs[0].substring(0,len);
        
    }
}

File: Dataset/Leetcode/valid/62/279.py
 class Solution:
    def XXX(self, m: int, n: int) -> int:
        dp = [1 for _ in range(n)]
        for row in range(1, m):
            for col in range(1,n):
                dp[col] = dp[col] + dp[col-1]
        return dp[-1]

